frontend/components/commons/forms/questions-form/fields/rating/Rating.component.vue

```diff
@@ -44,8 +44,8 @@ export default {
   methods: {
     onChangeRating(newOptions) {
       this.$emit("on-change-rating", newOptions);
-      const isAnyRatingSelected = this.isAnyRatingSelected(newOptions);
 
+      const isAnyRatingSelected = this.isAnyRatingSelected(newOptions);
       if (this.isRequired) {
         this.$emit("on-error", !isAnyRatingSelected);
       }
```

Comment:
Not sure to understand this logic. If the component is configured as required, the component will launch an `on-error` event together with an `on-change-rating`. For me, make sense to send one of the events. Let me know if I missing something.

Reply:
As discussed, this could be refactor with a v-model later. Also the form object at the parent level have to be update to reflect the current value from nested form-input components